src/main/java/core/basesyntax/service/ReaderService.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.FruitTransactionDao;
+import core.basesyntax.dao.FruitTransactionDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public class ReaderService {
+    private static final String FILE_NAME = "input.csv";
+    private FruitTransactionDao fruitTransactionDao = new FruitTransactionDaoImpl();
+
+    public List<FruitTransaction> readFromFile() {
```

Comment:
Create method for reading only one file reading, make it universal and flexible

Reply:
> Create method for reading only one file reading, make it universal and flexible

not quite understand what you mean